### PR TITLE
vary error for callback vs promise

### PR DESCRIPTION
**Pull request by bruce-lindsay**
https://github.com/cucumber/cucumber-js/issues/1019

### CODE REVIEW

File: features/step_definition_timeouts.feature [hunk 1]
```diff
@@ -81,7 +81,7 @@ Feature: Step definition timeouts
     Then it fails
     And the output contains the text:
       """
-      function timed out after 500 milliseconds
+      function timed out
```
Comment (charlierudolph) [inline features/step_definition_timeouts.feature]: Can we do what was done earlier in this file?

File: features/before_after_all_hook_timeouts.feature [hunk 2]
```diff
@@ -29,7 +29,8 @@ Feature: before / after all hook timeouts
     Then it fails
     And the error output contains the text snippets:
       | a handler errored, process exiting        |
-      | function timed out after 500 milliseconds |
+      | function timed out                        |
+      | 500 milliseconds                          |
```
Comment (charlierudolph) [inline features/before_after_all_hook_timeouts.feature]: Can we update this to: `function timed out, ensure the callback is executed within 500 milliseconds`?

**Comment by bruce-lindsay** [comment]
@charlierudolph sure thing. Glad to be able to help. I addressed your feedback in the latest commit. I thought I was being smart by reducing some of the repetitiveness but this is more consistent.  Let me know if I missed anything else.

**Comment by aslakhellesoy** [comment]
Hi @bruce-lindsay,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Pull request merged**